ResearchJournal.tex

```diff
@@ -0,0 +1,101 @@
+
+\subsection*{Heuristics}
+
+The Oxford dictionary defines Heuristics in regards to computing as proceeding to a solution by trial and error or by rules that are only loosely defined.  A* uses a weighting for each tile defined as the sum of the distance from the start node, and the distance from the target as a heuristic (H), to find the path of least cost. Commonly written as finding the smallest value of $f(n) = g(n) + h(n)$.  
+
+The Manhattan distance is a well known admissible heuristic, and is defined as $\Delta x + \Delta y$ \cite{botea2013pathfinding} which can be seen on an obstacle free grid as the direct route and distance.  The octile distance is a variant of the Manhattan distance adapted to 8-connected maps. It is defined as $\sqrt{2} \cdot m + (M - m)$, where $m = min(\Delta x, \Delta y)$ and $M = max(\Delta x, \Delta y)$ \cite{botea2013pathfinding}, essentially examining the hypotenuse of an equilateral right angled triangle with sides of length $m$.  $\sqrt{2}$ features prominently in square grid pathfinding although it is often approximated.  The degree of accuracy should be highly pertinent, although for short paths, $\sqrt{2} \approx 1.4$ can be sufficiently accurate.
+
+\subsection*{Optimizations}
+
+Given the success and popularity of A* pathfinding, there has been considerable effort to improve and refine it.  Typical speed-up enhancements include reducing the size of the search space using abstraction, and building more informed heuristics \cite{harabor2010breaking}, although the execution time, memory overhead, and whether the environment of the search system is static, dynamic, or real-time \cite{hart1968formal} (working under the assumption that there is a limited amount of time to make a move) must be considered.  Specific per case optimization is most important. For instance, a slightly overestimating heuristic increases the speed at the expense of accuracy \cite{millington2016artificial}, or were the map not to feature dead ends that would require back tracking then using the node with the best heuristic as far as a goal node is favourable \cite{togelius20102009}.
+
+Dynamically changing environments \cite{botea2011ultra} are a common occurrence in games and A* has difficulty in solving the problem of random dynamics obstacles avoidance \cite{wang2012game}.  Sometimes we just want to start the movement in the correct direction before its ultimate outcome is known, as replanning from scratch every time the graph changes can be very computationally expensive \cite{ferguson2005guide}.  Pathfinding in a smaller space abstraction can reduce the memory footprint \cite{lawrence2013database} especially when the end node of the solution is subject to change.  A lookahead search is also effective for such problems \cite{korf1990real}, but although looking ahead can improve the solution quality, suboptimal solutions can be produced even with admissible and consistent heuristics \cite{bulitko2003lookahead}.
```

Comment:
Good work getting this many citations into one paragraph! Exactly what I want to see.